### DISCUSSION TITLE
Installation ohne Funkmodul aber mit HM Lan Gateway

### DISCUSSION THREAD

**Discussion by MyUniversalDeveloper** [OpenCCU/OpenCCU#1162] [🇩🇪 Fragen&Antworten] [answered]
Hallo Jens,

ist es möglich eine RaspberryMatic Installation (z.B. unter Home Assistant) ohne Funkmodul aber mit einem HM Lan Gateway zu benutzen? Im Speziellen mit einer umkonfigurierten CCU2?

Danke für deine Arbeit und viele Grüße
Johannes

**Comment by jens-maus** [comment, marked as answer]
Ja, das ist möglich! Einzige Einschränkung ist: Ein "HM LAN Gateway" kann technisch bedingt nur traditionelles BidCos-RF/HomeMatic und **kein** homematicIP. D.h. mit so einer Lösung wird man nur "HomeMatic" Geräte ansteuern/nutzen können und keinerlei Geräte die aus dem "homematicIP" Bereich stammen.

**Comment by MyUniversalDeveloper** [comment]
Danke dir. Das HM IP nicht funktioniert ist bei mir noch kein Thema. Impliziet entnehme ich deiner Antwort, dass eine "umgebaute" CCU2 ebenfalls funktioniert. Ich hoffe ich liege richtig.

**Reply by jens-maus** [reply]
> Implizit entnehme ich deiner Antwort, dass eine "umgebaute" CCU2 ebenfalls funktioniert. Ich hoffe ich liege richtig.

Ja, auch eine als BidCos-RF LAN Gateway umgebaute CCU2 sollte mit einer RaspberryMatic ohne jegliches eigenes Funkmodul für einen reinen BidCos-RF/HomeMatic Funkbetrieb funktionieren.

**Comment by MyUniversalDeveloper** [comment]
Was für ein Service. Extrem schnelle Antwort. 👍 Beim nächsten Mal habe ich hoffentlich einen konstruktiven Beitrag. Mach's gut.

**Reply by horstvogel** [reply]
Moin, ähnliche Frage. Ich habe den **HmIP-RFUSB an der Home Assistent Integration** OS auf einem NUC, ich möchte nun den Empfangsbereich auf ein anderes Gebäude erweitern. Ich nutze nur HmIP Wireless Geräte. 
  or use a HmIP-HAP as a range extender for homematicIP.   
Mit dem HmIP-HAP kann ich das dann schon mal nicht. Gibt es eine andere Möglichkeit? Das Gebäude ist sehr weit weg, es besteht nur eine Netzwerkverbindung. Oder muss ich dann eine CCU dort platzieren und dann in die Home Assistant einbinden. Wäre natürlich schade, so kann ich die höhere Rechenleistung " vom Nuc nicht nutzen. 
Wenn ich mir die ganzen Github Einträge so anschaue, dann bist Du da dran das das funktionieren wird? Oder ist das noch ein sehr langer weg. So richtig verstehe ich das leider fachlich und sprachlich nicht. Vielen Dank für Deine Arbeit!!!

Danke!

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#known-limitations

**Comment by BenHei82** [comment]
Alten Thread ausgraben oder neuen beginnen… Da das Thema hier wirklich sehr passend ist und im übrigen auch das einzige in der Richtung ist was ich im Netz gefunden habe dachte ich ans ausgraben um auch künftigen Suchenden hier her zu „führen“.

ich habe Raspberrymatic ganz frisch auf einer Proxmox VM eingerichtet und in meinem Netzwerk habe ich ein HM-LGW-O-TW-T-EU der aktuell in FHEM im einer VCCU funktionierend eingebunden ist.

ich würde diesen aber gerne in Raspberrymatic einbinden - quasi als USB Ersatz (gerade wo die Positionierung viel einfacher ist. Ich weiss nur nicht wo ich ihn einrichte. Meine Vermutung war (oder ist):

Systemsteuerung/LAN-Gateway


Da hab ich die as/n, Passwort und IP hinzu gefügt und er erscheint in der oberen Liste ohne Status (aber auch ohne Fehlermeldung). Ein Neustart des RM hat recht lange gedauert:

post-Screen: LGFirmwareupdate (also wohl im Netzwerk gefunden und erkannt / warum sonst der Versuch?!)

ein anlernen meiner Geräte (HM Rollladenaktor) schlägt fehl da „keine Hatdware gefundener der man anmelden kann“. Natürlich hab ich meinen „Testaktor“ vorher resettet.

ich hoffe mir kann geholfen werden - Vielen Dank!

**Reply by Baxxy13** [reply]
Ich habe kein echtes HM-LGW-O-TW-T-EU zum testen.
Aber mit einer RaspberryMatic die zum LGW umfunktioniert wurde funktioniert das genau so.

- in der LAN Gateway Konfiguration auf hinzufügen klicken
- Name, Seriennummer und IP-Adresse eintragen (einen Zugriffscode hat mein RM-LGW nicht) und dann OK
- auf übernehmen klicken und dann rebooten

Danach ist das Teil connected und die beiden HM Anlernmodi verfügbar.

Zum Test einen HM Schaltaktor angelernt.
![RMLGW_Aktor angelernt](https://github.com/jens-maus/RaspberryMatic/assets/54898249/f9273a2f-9e21-4390-9ff1-63a79c522e19)

(RM als Homeassistant Add-On, reines Testsystem)

**Reply by BenHei82** [reply]
Okay, dein Screenshot hilft schonmal weiter - also dass er in der oberen Liste steht ist schonmal richtig - dachte, dass er unten erscheinen muss nach übernehmen (hab irgendwo im Netz ein Screenshot gesehen wo er unten war).

und du hast einen Statis „verbunden“ mein Statusfeld ist blank - entsprechend auch kein DutyCicle oder Firmware-Info. Kann ich das Gerät nur einmal anmelden - weil ich es ja derzeit noch in FHEM als VCCU nutze?

**Reply by BenHei82** [reply]
Moin nochmal, habe die Verbindung in FHEM mal testweise auf „closed“ gestellt und ein paar Sekunden später hatte ich ein „Verbunden“ in RM. Anlernprozess war recht einfach danach (auch wenn die Zuordnung der Räume ganz schön hin und her klicken ist - wäre es nicht praktisch dies direkt beim Testlauf in dem Posteingang zu erledigen?) 

Wir auch immer… danke für den Denkanstoß Baxxy13 👍🏽
